pandas/core/accessors.py

```diff
@@ -0,0 +1,345 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+
+An example/recipe for creating a custom accessor.
```

Comment:
@shoyer: You commented on this in #14781.  Have you found that custom accessors are frequently used by xarray users?

@jreback I can delete it or move it somewhere out of the way.  The `test_accessors` file might make a good place to park the discussion seeing as how it uses the same example anyway.